cmd/nitro/nitro.go

```diff
@@ -218,16 +218,16 @@ func mainImpl() int {
 		nodeConfig.Node.ParentChainReader.Enable = true
 	}
 
-	if nodeConfig.Node.Sequencer.Enable {
-		if nodeConfig.Node.ForwardingTargetF() != "" {
+	if nodeConfig.Execution.Sequencer.Enable {
```

Comment:
Should we have a check that `nodeConfig.Sequencer == nodeConfig.Execution.Sequencer.Enable`, or is this checked somewhere else already?